deploy/kustomize/daily/examples/watch-another-namespace/rco-deployment.yaml

```diff
@@ -11,3 +11,4 @@ spec:
           env:
             - name: WATCH_NAMESPACE
               value: rco-watched-ns2
+              valueFrom:
```

Comment:
Since this example builds upon overlay, which already removes the `valueFrom` field, it doesn't need to be specified here again.